auth0/resource_auth0_connection.go

```diff
@@ -550,6 +550,24 @@ var connectionSchema = map[string]*schema.Schema{
 					Optional:    true,
 					Description: "X.509 signing certificate (encoded in PEM or CER) you retrieved from the IdP, Base64-encoded",
 				},
+				"signing_key": {
+					Type:     schema.TypeList,
+					Optional: true,
+					MinItems: 1,
```

Comment:
```suggestion
					MinItems: 0,
```

This should probably be 0 if we're setting Optional as true.

Reply:
From the docs actually: 

```
// MinItems defines a minimum amount of items that can exist within a
// TypeSet or TypeList.
//
// If the field Optional is set to true then MinItems is ignored and thus
// effectively zero.
MinItems int
```

So we can remove the field completely.